epub33/core/index.html

```diff
@@ -6303,6 +6303,27 @@ <h3>Introduction</h3>
 					ensure the content will retain its integrity when rendered in a non-browser context.</p>
 			</section>
 
+			<section id="sec-nav-content-req">
+				<h3>Navigation document requirements</h3>
+
+				<p>A valid EPUB navigation document:</p>
+
+				<ul class="conformance-list">
+					<li>
+						<p id="confreq-navdoc-xhtml">MUST conform to the content conformance constraints for [=XHTML
+							content documents=] defined in <a href="#sec-xhtml-req"></a>;</p>
+					</li>
+					<li>
+						<p id="confreq-navdoc-navs">MUST conform to the <code>nav</code> element constraints defined <a
+								href="#sec-nav-def-model"></a>;</p>
+					</li>
+					<li>
+						<p id="confreq-navdoc-toc">MUST include exactly one <code>toc</code>
+							<code>nav</code> element as defined in <a href="#sec-nav-toc"></a>.</p>
+					</li>
```

Comment:
does it matter that it's &lt;code&gt;toc&lt;/code&gt; &lt;code&gt;nav &lt;/code&gt; rather than &lt;code&gt;toc nav&lt;/code&gt; ?